src/graphql/queries/patients.ts

```diff
@@ -2,6 +2,11 @@ import gql from 'graphql-tag';
 import { CCP } from './ccps';
 import { Hospital } from './hospitals';
 
+export enum Gender {
+  M = 'M',
+  F = 'F',
+}
+
```

Comment:
i'm curious what defining this here is doing - we're not importing it anywhere so is it being unused or implicitly being imported in the other files?

Reply:
Not sure if I'm understanding the question correctly but this is the enum we are using throughout all of the patient changes I made. In terms of why we are mapping it to strings 'M' and 'F', that's how it maps to the enum values on the backend.

Reply:
@petorsong `Gender` is being imported and used in `src/containers/PatientProfilePage.tsx` & `src/graphql/mutations/patients.ts`

Reply:
brainfart, my bad